src/components/contactUsForm/Styles.js

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components";
+import { RedButton } from "../globalStyles";
+
+export const FormContainer = styled.div`
+  max-width: 750px;
+  max-height: 350px;
```

Comment:
I always get a little wary when I see hard-coded widths and heights. I'm sure you know what you are doing here though.